engine/src/test/java/org/camunda/bpm/engine/test/bpmn/behavior/BpmnBehaviorLoggerTest.java

```diff
@@ -21,41 +21,48 @@
 import org.camunda.bpm.engine.test.Deployment;
 import org.camunda.bpm.engine.test.util.PluggableProcessEngineTest;
 import org.camunda.commons.testing.ProcessEngineLoggingRule;
+import org.junit.After;
 import org.junit.Rule;
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.*;
 
 public class BpmnBehaviorLoggerTest extends PluggableProcessEngineTest {
 
+  @After
+  public void tearDown() {
+    processEngineConfiguration.setEnableExceptionsAfterUnhandledBpmnError(false);
+  }
+
   @Rule
   public ProcessEngineLoggingRule processEngineLoggingRule = new ProcessEngineLoggingRule().watch(
       "org.camunda.bpm.engine.bpmn.behavior", Level.INFO);
 
   @Test
-  @Deployment(resources = { "org/camunda/bpm/engine/test/bpmn/behavior/UnhandledBpmnError.bpmn20.xml" })
-  public void testUnhandledBpmnErrorThrowException() {
+  @Deployment(resources = {
+      "org/camunda/bpm/engine/test/bpmn/behavior/BpmnBehaviorLoggerTest.UnhandledBpmnError.bpmn20.xml" })
+  public void shouldIncludeBpmnErrorMessageInUnhandledBpmnError() {
     // given
     processEngineConfiguration.setEnableExceptionsAfterUnhandledBpmnError(true);
     try {
       // when
       runtimeService.startProcessInstanceByKey("testProcess");
     } catch (ProcessEngineException e) {
       // then
-      assertTrue(e.getMessage()
-          .contains(
-              "Execution with id 'serviceTask' throws an error event with errorCode 'errorCode' and errorMessage 'ouch!', but no error handler was defined"));
+      assertThat(e.getMessage()).contains(
+          "Execution with id 'serviceTask' throws an error event with errorCode 'errorCode' and errorMessage 'ouch!', but no error handler was defined");
     }
```

Comment:
🔧 I noticed there is a better way to assert the exception using `assertThatThrownBy`. Check out [this test](https://github.com/camunda/camunda-bpm-platform/blob/736692fa374f4f711071d115e0684b1b7903493e/engine/src/test/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationTest.java#L87-L90) for an example.

Reply:
I have updated this test to use `assertThatThrownBy`. Please take a look!